### PR TITLE
Adding `heramb.dino.icu`

### PR DESCRIPTION
**Pull request by Heramb0x1**
# Adding `heramb.dino.icu`

## Description

- made all suggested changes.
- thanks for helping @alx-alexpark

### CODE REVIEW

File: dino.icu.yaml [hunk 1]
```diff
@@ -172,6 +172,12 @@ geta: # yoda - https://getadinoicu.yodacode.repl.co/dino.png - get a dino from h
   type: CNAME
   value: ef35bb48-f121-4f7f-91f6-93f601559c24.id.repl.co.
 
+heramb:
+  ttl: 600
+  type: CNAME
+  value: https://heramb-814z38mf5-heramb0x1s-projects.vercel.app.
```
Comment (alx-alexpark) [inline dino.icu.yaml:178]:
```suggestion
  value: cname.vercel-dns.com.
```

**Review by alx-alexpark — CHANGES_REQUESTED** [review]

**Comment by Heramb0x1** [comment]
Check it once @alx-alexpark !!

File: dino.icu.yaml [hunk 2]
```diff
@@ -172,6 +172,12 @@ geta: # yoda - https://getadinoicu.yodacode.repl.co/dino.png - get a dino from h
   type: CNAME
   value: ef35bb48-f121-4f7f-91f6-93f601559c24.id.repl.co.
 
+heramb:
+  ttl: 600
+  type: A
+  value: 76.76.21.21
+
```
Comment (alx-alexpark) [inline dino.icu.yaml:179]:
```suggestion
heramb:
  ttl: 600
  type: CNAME
  value: cname.vercel-dns.com.
```

**Review by alx-alexpark — CHANGES_REQUESTED** [review]

File: dino.icu.yaml (hunk 2 above)
Comment (Heramb0x1) [reply dino.icu.yaml]: did the suggested changes 👍  @alx-alexpark

**Review by alx-alexpark — APPROVED** [review]
lgtm

**Comment by Heramb0x1** [comment]
lfg!!
thanks mann!! @alx-alexpark

**Pull request merged**